tools/python/FindOptimizerOpsetVersionUpdatesRequired.py

```diff
@@ -30,8 +30,11 @@ def parse_args():
 
```

Comment:
Could you out of scope for this PR - but since we are updating the script, do you think it makes sense to update the file name to snake_case to follow python conventions?